notebooks/260-pix2struct-docvqa/260-pix2struct-docvqa.ipynb

```diff
@@ -0,0 +1,381 @@
SNb09mVdDwt4wwystuzatPgVXNy2+qofWUhX37a07WbYDNywYLJn55HORqhUNtRSu5XRde3ot4tQ7TTiPJhKTL3wmJYkLiZkZZY/c1BP5zXJJFPp6eVRXbw8tzps7D7OrBhJu3N37MQLTJtI3E8TVoHR9W6+u44f7aZUnxqULMJNmzR2QmDfRdGtb2c3Gkq1g7MMsuwxqo597Y+vOV0QYI93Ko09sOpPA8BzY1116yeB9vPdA+kcnysLvPM5kNm7qSoT3Gp6+7vTPYZ/45rL12dpaq/elTKJS1KB1cAtW6u3z+/5+kKsX2LmlwgkTqXbSHdSg/Zj+DtlX9l2S+bss1AZ9BnppfD638fBHmSEcLwrbu/WfbN3AQd1Eb19JZVTbwZiGTr69Jw1oQi1IiBc9ka6WoS6HmImnp4Ug4s5dKTde4p270YiNp4cBhrAaNXFBY16FShd8CLJCXseTt711s6OsVsTjeB/jyldXrlc91ZWcVq2PrAy195V6RU0JavpJNX+ukqPSNCy3Ji5Y4oO70cR1WLQJMu/eDucbeDSp8cJylUQleyjx8UWcU1kVs6jm9iVVq2wLoRahAhVFZHX4klpk2qfanVS7IjFVaGikhWekZpKDkSAz/F265AAz8iYGnIhrV2JJ9NJq1bS4ND8RQnn26m6fcXXbyrNvKC16diLfwaLYeXroV4befiT9AEr5+9sP06gNmzYiBjWV6skI/1a7VNemjTXynt0OLZVIxAuf3n5RrOne1JmKUGw93fWK377+JKVR8f7lWzInxb6Fj3H5hzfJujZyg6WNuZ6iiTGFGivtIIpyEy6EMjU1yeBYkPX4gcz3oShUKgWvrJSq+lWDObVBC2/ir0Vcflj1NxzkNMLsuvXyLL93cOnRR+UNu3cVXjoUbVxiBhzV1GSR19Ly8AcvpGZXVKAqjeinHub8iNdhJG5iMcjr0CyZC1tV3jrv1d2ZlVeh2p+VlVMTYI3iKocOlc5M3FXV8BCR+DoM+zXUUZlQY+xVkBsz9PSy4b65dPxFlr23JzGRRSfWpXBCz558na/XxNue9HUFJdVI+hY9VFk1wg/7oixN6QM6XvzDx+TnzZSVUZJe0y6qe7oSMV+ZTHXxbExPJ+Y9yRFEkP3k4Qflq2i+sro6FVcyvNRJhigzqtNq9Ej3F9vXjZ/7uX9QA2N6aUrYtZPno/SDlg0WrccRpJxbuOKdVVu/htYmepSimHuHH5UatvO2pSCYff9p/e9M3/jbtKSh3X3s9bDSvIzEd88/GAxbPcqthoIou93sFR9T5pyYNiy+Z79OTSw1uVkfH1w8/yCZ3WnJ7/1l3u/FtNPOzVkqGNNN+B2VkzvOZ9kPXR4kXGCBmnSZPPr6+D2TJ2YN6xPgbEQtz8tM/vTyOdJ18xRfaSivkgGx4JL4qz8Prp3JRA17Bzdnqcwse5LmPOCXoPPzb74VYKZkOmbRZ8niT5OW7Ptt+Kfevds2NmOUpYbf/uviizzb/mvmdTSsPt9AlON/PDB9Z3rjIB8XSyNtJPfjP8dCEatBX3mHTciluPb7NeifJTunL6r8tVsjrYLwi3sPRrI7rO4rmu+z7Doq+MycXStPWC/oT3xHJfTkyqMRxGMoyVY3O5KlhL90z+FTOjxYtHnCjMyRPX0smWVZsWHPKwN//8WbplIfWRnq7iv1ipoXQEyVn1CU+bOapsEsuo3pcX7G0blz0LH9vAzLo64d2PuC2nzeEC+1YxxpkzFTO1tWedjVC8/13HVpDENbe2OV3NSuWs22EIqgurb2Bvwbt/56bOtnSKfqWTmZq12LtB1yO6rdSbUrIjT3oNbsK5d2n2k5v7sDlv7y5IZTsRREMphg9v2m9L05/eDUSUUjB/jba3HyPr97muU2a2JrHVUWVzwyYHZde3keWfuqQL/r5DZVy7eYzYeN8rq3cd30VQUj2znQM1+c2n02w6b/gmDRUl+V6slR+CYHqH7Q6EGnfjv4+3TmuKH+loLPD47uvlHm9tuoVsI5BA2fIUMb3927dkujlRPaWAkSbm/ddKuAihiKq6Z7jpzW6fGipeMqowZ1bmqtxS/KSY97+zTJaerSvtJ3RRRpqXzAV5SbTKM4t/DWvXDz6KVOC3s4UlKfHF55NJJ4xExuVFsHa+Txg4v33DuYMylaFk7WunU3GSkM1Q4c+1vzl2tX/IZ8Ht7Zw4RSnB71Olyjx4KBruLwCDPu0Cdg9/xbkUz/xZ2U/dkKlN3Exwk9ePbQc69xzdkln/7ZtPZOMVPhTDtZcdUvMd6O9r/8xxbiCzUzg53o6U/3rTqfTJU+S6nK+AV7Kocvxc6svBbV/qysnFoAaxZWPTaqdGakpodoCytQOuzXrL1uKQrGXpmFYhJZFDuvJuz9Z18VWg8XT0xqe3o75m4K4Wm2/a3aZ9XqVrsw9xf30Nqrork1b8JYe+nw3cAZQebchDs7l11Il1n8UbuAqhw17GKluqdXlfyWeyi77Yh+x8efXLiINbGfN/EdlesHTsWxiHVeVWHMt6yuTrJqhLp1Ki2Xme487M+9hgf2n729949jxQK6vkWD5mM3jRvaUvwIBtVxdDN/fOvMtr8y88pwlrGDV68Vv//qJ3ybCNX2nrpnr/2BA38dX3Uuq4KibWBq6+bbUdnbkKi+z5TdRz1PHzpz6/TmS/mVFG1TJ68+S5YND3aVWQdELPFqQnwi8d2W7fN2ZnC0bFuM3jDrF/LNJobryK0HLQ/vO3V+07X0Ypylb2zdoHmbvsSNgdob8Vd/WhvcvJhr1q6zZ82FzsrFoHqtfxnoen+XzGQPMVKYtl100KrZ8SMX7h1Zd7aIS/whKVefEWtGDmljJ70nlROJGrq66YU+PLbpZFZBOaZt7tJ85PqJwxvWdYSVkyk6wIw7Lt1D27/t0F+rZ+6uZBo7+QxfP3l0oPgVNOIOZM72ZTs27J3Vc2eFhknD1kPm/CKYs5uUUkc7ksWIX8yo3e/7tRx3Hby8a/HxQj7TwM6
```

Comment:
Looks complicated for this tutorial. It is always URL starting with `https`

Reply:
simpilfied